sdk/templates/go/README.mustache

```diff
@@ -0,0 +1,235 @@
+# Go API client for {{packageName}}
```

Comment:
You mentioned that these templates are downloaded. Out of curiosity, where do they come from? If we needed to update them, how would we do so?

Reply:
The command line outputs them `openapi-generator-cli author template -g typescript-axios -o ./typscript `